admin/services/data-access/setting.data.service.js

```diff
@@ -1,11 +1,24 @@
 'use strict'
 
 const Setting = require('../../models/setting')
+const sqlService = require('./sql.service')
+const TYPES = require('tedious').TYPES
 
 const settingDataService = {}
 
-settingDataService.findOne = async function (options) {
-  return Setting.findOne(options).lean().exec()
+settingDataService.findOne = async function () {
```

Comment:
Can we call new the sql methods s`sqlFindOne()`?  Then we can build them up side by side and then delete the existing mongo methods.

We can rename from sqlFindOne to findOne at the end.